samri/plotting/examples.py

```diff
@@ -231,9 +244,9 @@ def seed_connectivity_overview(
 	# 	overlays=["~/ni_data/templates/roi/DSURQEc_dr.nii.gz",],
 	# 	)
 
-def functional_connectivity(ts,
-	brain_mask="~/ni_data/templates/DSURQEc_200micron_mask.nii.gz",
-	labels = '~/ni_data/templates/roi/DSURQE_mapping.csv',
+def functional_connectivity(ts="/Users/marksm/GitHub/mriPipeline/restingState/von_christian/a.nii.gz",
+	brain_mask="/Users/marksm/GitHub/mriPipeline/restingState/von_christian/labels_originals.nii",
+	labels = '/Users/marksm/GitHub/mriPipeline/restingState/von_christian/labels.csv',
```

Comment:
We need to develop a more unified datapath - are you sure you can't keep your SAMRI-related data under `/Users/marks/ni_data/` (I assume under MacOS, `~` evaluates to `/Users/marks`). If not, could you at least set up a symlink?

Ideally, further down the line, we will be [distributing example data in a standardized path](https://github.com/IBT-FMI/SAMRI/issues/25) for the examples, but until then, I think the best approach, is for all developers to use analogous data paths.